src/Markdig/Parsers/BlockProcessor.cs

```diff
@@ -617,12 +617,6 @@ private void TryContinueBlocks()
                     {
                         leaf.AppendLine(ref Line, Column, LineIndex, CurrentLineStartPosition);
                     }
-
```

Comment:
This is a safe guard that should not be removed. This was put in purpose so that if an extension forgot to add a NewBlock, we throw an error.